tempest/services/identity/xml/identity_client.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2012 IBM Corp.
```

Comment:
I think we should not add any of these files on our own. Let tempest take care of it?